source/simulator/assembly.cc

```diff
@@ -984,6 +1012,14 @@ namespace aspect
 
 
 
+  template <int dim>
+  Simulator<dim>::Assemblers::Properties::Properties ()
+    :
+    need_face_material_model_data (false)
```

Comment:
do you need to initialize ```needed_update_flags`` here? Maybe this causes the tests to fail. The error is
```
An error occurred in line <3397> of file </root/deal.II/dealii/source/fe/fe_values.cc> in function
    void dealii::FEValues<dim, spacedim>::initialize(dealii::UpdateFlags) [with int dim = 2; int spacedim = 2]
The violated condition was: 
    (update_flags & update_normal_vectors) == false
The name and call sequence of the exception was:
    typename FEVB::ExcInvalidUpdateFlag()
Additional Information: 
(none)
```